detectable/src/test/java/com/synopsys/integration/detectable/detectables/bazel/functional/bazel/pipeline/WorkspaceRuleChooserTest.java

```diff
@@ -24,55 +24,48 @@ class WorkspaceRuleChooserTest {
 
     @Test
     void testOneRuleParsed() throws IntegrationException {
-        Set<WorkspaceRule> chosenWorkspaceRules = run(null, null, WORKSPACE_RULES_JUST_MAVEN_INSTALL);
+        Set<WorkspaceRule> chosenWorkspaceRules = run(null, WORKSPACE_RULES_JUST_MAVEN_INSTALL);
         assertEquals(1, chosenWorkspaceRules.size());
         assertEquals("maven_install", chosenWorkspaceRules.iterator().next().getName());
     }
 
     @Test
     void testThreeRulesParsed() throws IntegrationException {
-        Set<WorkspaceRule> chosenWorkspaceRules = run(null, null, WORKSPACE_RULES_THREE);
+        Set<WorkspaceRule> chosenWorkspaceRules = run(null, WORKSPACE_RULES_THREE);
         assertEquals(3, chosenWorkspaceRules.size());
     }
 
     @Test
     void testOneProvidedSameOneParsed() throws IntegrationException {
-        Set<WorkspaceRule> chosenWorkspaceRules = run(null, WORKSPACE_RULES_JUST_MAVEN_INSTALL, WORKSPACE_RULES_JUST_MAVEN_INSTALL);
-        assertEquals(1, chosenWorkspaceRules.size());
-        assertEquals("maven_install", chosenWorkspaceRules.iterator().next().getName());
-    }
-
-    @Test
-    void testOneProvidedViaDeprecatedPropertySameOneParsed() throws IntegrationException {
-        Set<WorkspaceRule> chosenWorkspaceRules = run(WORKSPACE_RULES_JUST_MAVEN_INSTALL, null, WORKSPACE_RULES_JUST_MAVEN_INSTALL);
+        Set<WorkspaceRule> chosenWorkspaceRules = run(WORKSPACE_RULES_JUST_MAVEN_INSTALL, WORKSPACE_RULES_JUST_MAVEN_INSTALL);
         assertEquals(1, chosenWorkspaceRules.size());
         assertEquals("maven_install", chosenWorkspaceRules.iterator().next().getName());
     }
 
     @Test
     void testPreferNewProperty() throws IntegrationException {
-        Set<WorkspaceRule> chosenWorkspaceRules = run(WORKSPACE_RULES_HASKELL, WORKSPACE_RULES_JUST_MAVEN_INSTALL, WORKSPACE_RULES_JUST_MAVEN_INSTALL);
+        Set<WorkspaceRule> chosenWorkspaceRules = run(WORKSPACE_RULES_JUST_MAVEN_INSTALL, WORKSPACE_RULES_JUST_MAVEN_INSTALL);
         assertEquals(1, chosenWorkspaceRules.size());
         assertEquals("maven_install", chosenWorkspaceRules.iterator().next().getName());
     }
```

Comment:
I think the need for testPreferNewProperty() has gone away, and that test could be removed.